scripts/test/updateAtlas.js

```diff
@@ -22,7 +22,7 @@ async function main() {
         throw new Error("Cannot find a tests/testProject. Did you run the script in the widget folder?");
     }
 
-    rm("-rf", "tests/testProject/theme", "tests/testProject/themesource");
+    rm("-rf", "tests/testProject/theme", "tests/testProject/themesource/atlas_ui_resources");
```

Comment:
atlas module will be renamed, please stick to the initial suggestion. `rm("-rf", "tests/testProject/theme", "tests/testProject/themesource/atlas*");`

Reply:
Okay, changed it.